Sazabi.cpp

```diff
@@ -389,7 +389,7 @@ BOOL CSazabi::InitFunc_SGMode()
 		CString strCommandC;
 		CString strParam;
 
-		strParam.Format(_T("\"%s\" %s"), (LPCTSTR)m_AppSettings.GetRootPath(), (LPCTSTR)m_strConfigParam);
+		strParam.Format(_T("\"%s\" %s"), (LPCTSTR)m_AppSettings.GetRootPath().TrimRight(_T("\\")), (LPCTSTR)m_strConfigParam);

```

Comment:
It might be better to explain "why" here.
e.g. mention https://github.com/ThinBridge/Chronos/pull/300 and add a brief description.